### PR TITLE
clarify SL set up permissions

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr clarifies the permissions needed to configure the dbt SL per [slack thread](https://dbt-labs.slack.com/archives/C03NEJAEX1N/p1692377100089349)

### CODE REVIEW

File: website/snippets/_new-sl-setup.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must have the correct license and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
- You must have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```

File: website/snippets/_new-sl-setup.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```

**Review by mirnawong1 — COMMENTED** [review]
clarify

File: website/snippets/_new-sl-setup.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
+    * Enterprise plan &mdash; Developer license with Project Creator or Account Admins permissions
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator or Account Admins permissions.
```

**Review by mirnawong1 — COMMENTED** [review]
More clarification

File: website/snippets/_new-sl-setup.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
+    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator or Account Admins permissions.
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, or Account Admins permissions.
```

**Review by mirnawong1 — COMMENTED** [review]
Add

File: website/snippets/_new-sl-setup.md [hunk 5]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
+    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, or Account Admins permissions.
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, or Account Admin permissions.
```

File: website/snippets/_new-sl-setup.md [hunk 6]
```diff
@@ -1,5 +1,9 @@
...
+    * Team plan &mdash; Developer license with Owner permissions
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Team plan &mdash; Owner with a Developer license.
```

**Review by mirnawong1 — COMMENTED** [review]
clarify

File: website/snippets/_new-sl-setup.md [hunk 7]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
+    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, or Account Admin permissions.
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, Admin, or Account Admin permissions.
```

**Review by mirnawong1 — COMMENTED** [review]
Add admin

File: website/snippets/_new-sl-setup.md [hunk 8]
```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```
Comment (rpourzand) [inline website/snippets/_new-sl-setup.md:4]: I think the only thing to change here is that if you are an Account Admin, you actually don't have to be an Owner. You can just be "Everyone" in this case

**Review by rpourzand — CHANGES_REQUESTED** [review]
Changes noted in comments

File: website/snippets/_new-sl-setup.md [hunk 9]
```diff
@@ -1,5 +1,9 @@
...
+    * Enterprise plan &mdash; Owner with a Developer license and assigned Project Creator, Database Admin, Admin, or Account Admin permissions.
```
Comment (mirnawong1) [inline website/snippets/_new-sl-setup.md]:
```suggestion
    * Enterprise plan &mdash; Developer license with Account Admin permissions. Or Owner with a Developer license, assigned Project Creator, Database Admin, or Admin permissions.
```

**Review by mirnawong1 — COMMENTED** [review]
clarify acct admin

**Review by rpourzand — APPROVED** [review]
lgtm!

**Pull request merged**